package.json

```diff
@@ -16,15 +16,15 @@
   "homepage": "https://www.rungalileo.io/",
   "dependencies": {
     "axios": "^1.6.7",
-    "jsonwebtoken": "^9.0.2"
+    "jsonwebtoken": "^9.0.2",
+    "@types/jsonwebtoken": "^9.0.6"
```

Comment:
Types are a dev dependency and should not be in our global ones.

Reply:
Agreed, this doesn't make sense to me...the client gets transpiled from TS to JS during build, so there shouldn't be any reason we need the types after build time.

Reply:
@ktalebian requested this per their needs - maybe he can offer more here.